### PR TITLE
FE server common: move from pkg:usage to pkg:uuid

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Review by elliette — APPROVED** [review]

**Comment by elliette** [comment]
I believe this change has caused the test failures, do you mind reverting it @kevmoo ? Thanks!

**Comment by kevmoo** [comment]
Revert is here: https://github.com/dart-lang/webdev/pull/2414

**Comment by kevmoo** [comment]
@elliette the revert has the EXACT same failures. I think something else is going on!

File: frontend_server_common/pubspec.yaml [hunk 1]
```diff
@@ -17,4 +17,4 @@ dependencies:
   pub_semver: ^2.1.1
   test_common:
     path: ../test_common
-  usage: ^4.0.0
+  uuid: ^3.0.7
```
Comment (devoncarew) [inline frontend_server_common/pubspec.yaml:20]: Note that `uuid` is not currently dep'd into the sdk rep. We have - in one other location - chosen to in-line a uuid v4 implementation (but I don't currently re-call where).

`uuid` itself may be harmless to bring into the sdk; we do generally try to limit deps there (or, to be conservative about bring in new ones).

Comment (kevmoo) [reply frontend_server_common/pubspec.yaml:20]: Or we could just copy-paste. 🤷

Comment (devoncarew) [reply frontend_server_common/pubspec.yaml:20]: https://github.com/dart-lang/webdev/pull/2415

**Pull request merged**